### PR TITLE
shared/cfg/guest-os/Windows.cfg: Add some config for migrate_multi_host

### PR DESCRIPTION
**Pull request by spcui**
Signed-off-by: Shuping Cui <scui@redhat.com>

### CODE REVIEW

File: shared/cfg/guest-os/Windows.cfg [hunk 1]
```diff
@@ -66,6 +66,47 @@
...
     migrate.with_file_transfer:
         guest_path = C:\tmpfile
+    migrate_multi_host.with_stress, migrate_multi_host.set_downtime:
+        bg_stress_test = win_heavyload
+        timeout = 1800
+        autostress = yes
+        install_cmd = "start /wait DRIVE:\HeavyLoadSetup.exe /verysilent"
+        config_cmd = 'setx -m path "%PATH%;${install_path};"'
+        kill_bg_stress_cmd = "taskkill /T /F /IM heavyload.exe"
+        i386, i686:
```
Comment (xutian) [inline shared/cfg/guest-os/Windows.cfg]: move guest related config to guest-os.cfg

File: shared/cfg/guest-os/Windows.cfg [hunk 2]
```diff
@@ -66,6 +66,47 @@
...
+            install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
+    migrate_multi_host.virtio_blk_data_plane:
+        bg_stress_test = fio_windows
+        timeout = 300
+        i386, i686:
+            install_path = "C:\Program Files (x86)\fio"
+            install_cmd = "msiexec /i DRIVE:\fio-2.2.10-x86.msi /qn"
+            #download_url = http://www.bluestop.org/fio/releases/fio-2.2.10-x64.msi
+            #pkg_md5sum = "fa45dee52e1f45e048e0cee2e14bdcd6"
+        x86_64:
```
Comment (xutian) [inline shared/cfg/guest-os/Windows.cfg]: same as above lines

File: shared/cfg/guest-os/Windows.cfg [hunk 3]
```diff
@@ -66,6 +66,47 @@
...
+        config_cmd = 'setx -m path "%PATH%;${install_path};"'
+        start_cmd = '"${install_path}\fio.exe" --name=fiotest --rw=randread --iodepth=4 --bs=4k --ioengine=windowsaio --size=1g --numjobs=4'
+        kill_bg_stress_cmd = "taskkill /T /F /IM fio.exe"
+    migrate_multi_host.with_module_reload:
+        driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+        driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+        driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+        i386, i686:
+            devcon = "xcopy %s:\devcon\wnet_x86\devcon.exe C: /Y"
+        x86_64:
```
Comment (xutian) [inline shared/cfg/guest-os/Windows.cfg]: same as above lines

Comment (xutian) [inline shared/cfg/guest-os/Windows.cfg]: x86_64  go into  guest-name/x86_64.cfg and move nic related cfg into guest-hw.cfg. thanks

**Comment by spcui** [comment]
@xutian updated, could you review again?

**Comment by xutian** [comment]
looks good to me, merged thanks @spcui

**Pull request merged**